code/datums/mood_events/generic_negative_events.dm

```diff
@@ -114,10 +114,29 @@
 	mood_change = -4
 
 /datum/mood_event/healsbadman
-	description = "<span class='nicegreen'>You feel a lot better, but wow that was disgusting.</span>\n" //when you read the latest felinid removal PR and realize you're really not that much of a degenerate
+	description = "<span class='warning'>I feel a lot better, but wow that was disgusting.</span>\n" //when you read the latest felinid removal PR and realize you're really not that much of a degenerate
 	mood_change = -4
 	timeout = 1200
 
+/datum/mood_event/jittery
+	description = "<span class='warning'>I'm nervous and on edge and I can't stand still!!</span>\n"
+	mood_change = -2
+
+/datum/mood_event/vomit
+	description = "<span class='warning'>I just threw up. Gross.</span>\n"
+	mood_change = -2
+	timeout = 1200
```

Comment:
Welp, flypeople are gonna be pretty much perma-depressed now, as they need to vomit to eat (maybe they can inject themselves with nutriment? I'm not sure).

Reply:
it's a pretty negligible malus but I guess I'll add an exception for flypeople